pkgs/servers/lenpaste/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitea }:
+
+buildGoModule rec {
+  pname = "lenpaste";
+  version = "1.3";
+
+  src = fetchFromGitea {
+    domain = "git.lcomrade.su";
+    owner = "root";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-d+FjfEbInlxUllWIoVLwQRdRWjxBLTpNHYn+oYU3fBc=";
+  };
+
+  vendorSha256 = "sha256-PL0dysBn1+1BpZWFW/EUFJtqkabt+XN00YkAz8Yf2LQ=";
+
+  ldflags = [
+    "-w"
+    "-s"
+    "-X main.Version=${version}"
+  ];
+
+  subPackages = [ "cmd" ];
+
+  postInstall = ''
+    mv $out/bin/cmd $out/bin/lenpaste
+  '';
+
+  meta = with lib; {
+    description = "Open source analogue of pastebin.com. There is its own API.";
+    homepage = "https://git.lcomrade.su/root/lenpaste";
+    license = licenses.agpl3Plus;
+    maintainers = with lib.maintainers; [ vector1dev ];
```

```suggestion
    maintainers = with maintainers; [ vector1dev ];
```

Nit (because `maintainers` is already in the scope due to the `with lib;` above.
Alternatively, drop the `with lib;` and replace `licenses.agpl3Plus;` with `lib.licenses.agpl3Plus;`